File: assignment-18/task2.java
public class task2 {
    /**
     * Check if a number is positive, negative, or zero.
     * @param num the number to check
     */
    public static void checkNumber(int num) {
        if (num > 0) {
            System.out.println("The number is positive");
        } else if (num < 0) {
            System.out.println("The number is negative");
        } else {
            System.out.println("The number is zero");
        }
    }

    public static void main(String[] args) {
        System.out.println("Java Output:");
        checkNumber(-5);
        checkNumber(0);
        checkNumber(7);
    }
}

File: assignment-18/task2.py
def check_number(num):
    """Check if a number is positive, negative, or zero."""
    if num > 0:
        print("The number is positive")
    elif num < 0:
        print("The number is negative")
    else:
        print("The number is zero")


# Call the function with different inputs
print("Python Output:")
check_number(-5)
check_number(0)
check_number(7)
